test/command/suite/tokenizers/mecab/one_token_with_middle_space.test

```diff
@@ -0,0 +1,6 @@
+#@require-env MECAB_DICTIONARY_NAIST_JDIC
```

Comment:
I'm feeling strange that this test is executed whether `MECAB_DICTIONARY_NAIST_JDIC` is `YES` or `NO`.
What do you think?

Reply:
I understood the point. If we want to control the behavior using the specified value, I think we couldn't use this option.
I think it's no too strange because  `require-env` option is checking whether the specified environment variable is defined or not. Should we change the value not like `YES`?

Reply:
In CI, where `MECAB_DICTIONARY_NAIST_JDIC=YES` is set, it may be enough to add a comment such as “The value is meaningless and will be executed if this variable is defined”.

Anyway, I'm worried that if I see `MECAB_DICTIONARY_NAIST_JDIC=YES` it looks like I can disable it with `MECAB_DICTIONARY_NAIST_JDIC=NO`.

Reply:
@abetomo 
fix [9ef2d49](https://github.com/groonga/groonga/pull/2331/commits/9ef2d494c3ad58dd603b5eff3e5b66357da129e3): Thanks for the suggestion! At now, I’ve updated the CI step to include the comment clarifying that only the presence of `MECAB_DICTIONARY_NAIST_JDIC` is checked and its value is ignored. Next, I will extend grntest with a new option (e.g. --mecab-dictionary) in the following PR.